OracleUnifiedDirectorySM/dockerfiles/12.2.1.4.0/README.md

```diff
@@ -0,0 +1,116 @@
+Building an Oracle Unified Directory Services Manager Image with Dockerfiles and Scripts
+========================================================================================
+
+## Contents
+
+1. [Introduction](#1-introduction)
+2. [Hardware and Software Requirements](#2-hardware-and-software-requirements)
+3. [Build Oracle JDK/Oracle FMW Infrastructure image](#3-build-oracle-jdkoracle-fmw-infrastructure-image)
+4. [Building OUDSM Docker image](#4-building-oudsm-docker-image)
+
+# 1. Introduction
+This project offers scripts to build an Oracle Unified Directory Service Manager image based on 12cPS4 (12.2.1.4.0) release. Use this image to facilitate installation, configuration, and environment setup for DevOps users. 
+
+This image refers to binaries for OUD Release 12.2.1.4.0 and it has the capability to create an FMW Infrastructure domain with the OUDSM application deployed in a container with OUDSM deployed which can be targeted for development and testing.
+
+***Image***: `oracle/oudsm:12.2.1.4.0`
+
+# 2. Hardware and Software Requirements
+Oracle Unified Directory Service Managed Services image has been tested and is known to run on following hardware and software:
+
+## 2.1 Hardware Requirements
+
+| Hardware  | Size  |
+| :-------: | :---: |
+| RAM       | 16GB  |
+| Disk Space| 200GB+|
+
+## 2.2 Software Requirements
+
+|       | Version                        | Command to verify version |
+| :---: | :----------------------------: | :-----------------------: |
+| OS    | Oracle Linux 7.3 or higher     | more /etc/oracle-release  |
+| Docker| Docker version 18.03 or higher | docker version            |
+
+# 3. Build Oracle JDK/Oracle FMW Infrastructure image
+
+To build an OUDSM image either you can start from building the Oracle JDK and Oracle Fusion Middleware Infrastrucure image or use the already available Oracle JDK/Oracle Fusion Middleware Infrastructure image. The images are available in the [Oracle Container Registry](https://container-registry.oracle.com), and can be pulled from there. Steps to obtain the images from the [Oracle Container Registry](https://container-registry.oracle.com), or to build the images are provided in the steps below:
+
+## 3.1 Pulling the Oracle JDK (Server JRE) base image
```

Comment:
Please remove this. We only document how to build images in this repo, not pull them.

Reply:
@Djelibeybi as a very remote bystander to this PR I've agreed 100% with you about removing empty files and other nicenesses, but regarding the instructions here I think it does help users to have a single set of instructions they can follow.

Reply:
How about a comprise? Split the README into multiple files and link to three docs:

- Building all the images manually 
- Pulling the parent images and just building the OUD images manually 
- Using the WebLogic image tool to build the images

I'm trying to reduce the length and complexity of this README and that should be acceptable to all.

Reply:
Can you please provide some more details about why pulling instructions cannot be mentioned along with reference to OracleJava and OracleFMWInfrastructure? Customers looking for building OUDSM image, why can't they use pre-built Java and FMWInfrastructure images from OCR as base layer for OUDSM image ?

Reply:
Two reasons:
1. This repo is all about providing the tools and examples to _build_ images, not pull them.
2. I don't want duplicated documentation that could go out of date

In this particular case, your README.md files in this repo should link to the build instructions for the required images so that those instructions are not duplicated and maintained by the owner(s) of that image. Likewise, the installation guide for your product should have a section called "Installation in a container" or "Installation using Kubernetes" which documents how to pull the image(s) from  Oracle Container Registry.

So, if you want some golden guidelines: don't provide documentation for products you don't own and don't duplicate documentation from products you don't own. Just document your stuff, because that will be more than enough work (trust me on this).